Bottom: 33dbffcb180ed0ce33c0c7a5d45b842ed7c85ad1
Top:    5a1dc0ad4e48638a32ff60dcbe1195c5a5e107a5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 15:27:08 +0100

Refresh of hid-gtk-use-gtk_widget_get_sty

---

diff --git a/src/hid/gtk/gui-drc-window.c b/src/hid/gtk/gui-drc-window.c
index e7f5c46..ce9a564 100644
--- a/src/hid/gtk/gui-drc-window.c
+++ b/src/hid/gtk/gui-drc-window.c
@@ -719,6 +719,7 @@ ghid_violation_renderer_render (GtkCellRenderer      *cell,
 				GtkCellRendererState  flags)
 {
   GdkDrawable *mydrawable;
+  GtkStyle *style = gtk_widget_get_style (widget);
   GhidViolationRenderer *renderer = GHID_VIOLATION_RENDERER (cell);
   GhidDrcViolation *violation = renderer->violation;
   int pixmap_size = VIOLATION_PIXMAP_PIXEL_SIZE - 2 * VIOLATION_PIXMAP_PIXEL_BORDER;
@@ -751,7 +752,7 @@ ghid_violation_renderer_render (GtkCellRenderer      *cell,
 
   mydrawable = GDK_DRAWABLE (violation->pixmap);
 
-  gdk_draw_drawable (window, widget->style->fg_gc[GTK_WIDGET_STATE (widget)],
+  gdk_draw_drawable (window, style->fg_gc[GTK_WIDGET_STATE (widget)],
 		     mydrawable, 0, 0,
 		     cell_area->x + cell_area->width + VIOLATION_PIXMAP_PIXEL_BORDER,
 		     cell_area->y + VIOLATION_PIXMAP_PIXEL_BORDER, -1, -1);
